guides/source/caching_with_rails.md

```diff
@@ -512,13 +512,19 @@ class ProductsController < ApplicationController
 end
 ```
 
-### A note on weak ETags
-
-Etags generated by Rails are weak by default. Weak etags allow semantically equivalent responses to have the same etags, even if their bodies do not match exactly. This is useful when we don't want the page to be regenerated for minor changes in response body. If you absolutely need to generate a strong etag, it can be assigned to the header directly.
-
-```ruby
-  response.add_header "ETag", Digest::MD5.hexdigest(response.body)
-```
+### Notes on strong and weak ETags
+Strong ETag sets a "strong" ETag validator on the response. Strong ETag implies that
```

Comment:
I know this is describing what the `#strong_etag=` method and `strong_etag:` option do, but it sounds circular: Strong ETag sets a strong ETag 😁

Let's skip this and jump straight into discussion of when you need a strong ETag and how to get one.